crates/iota-sdk-ffi/src/types/crypto/move_authenticator.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::sync::Arc;
+
+use crate::types::{
+    address::Address,
+    digest::Digest,
+    object::{ObjectId, ObjectReference},
+    transaction::Input,
+    type_tag::TypeTag,
+};
+
+/// MoveAuthenticator is a signature variant that enables a new
+/// method of authentication through Move code.
```

Comment:
```suggestion
/// MoveAuthenticator is a signature variant that enables a
/// method of authentication through Move code.
```
probably shouldn't say "new"

Reply:
https://github.com/iotaledger/iota-rust-sdk/pull/442/commits/2facb0c482609a9012a2341c1d9e51b3599cb48f